common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1043,7 +1043,7 @@ def test_reset_problem_not_done(self):
 
     def test_rescore_problem_correct(self):
 
-        module = CapaFactory.create(attempts=1, done=True)
+        module = CapaFactory.create(attempts=0, done=True)
```

Comment:
Why did the value of number of `attempts` need to change?

Reply:
Also curious about this.

Reply:
I am submitting the problem first, that would increment the attempts to 1 so I changed to 0 initially. We need to submit the problem now to set the score as the rescore method does not set the score any more.

Reply:
Makes sense. Thanks @asadazam93